qt/python/mantidqt/widgets/fitpropertybrowser/fitpropertybrowser.py

```diff
@@ -222,13 +222,9 @@ def clear_fit_result_lines(self):
         """
         Delete the fit curves.
         """
-        for lin in self.fit_result_lines:
-            try:
-                lin.remove()
-            except ValueError:
-                # workspace replacement could invalidate these references
-                pass
-        self.fit_result_lines = []
+
+        if self.fit_result_ws_name:
+            self.get_axes().remove_workspace_artists(mtd[self.fit_result_ws_name])
```

Comment:
I believe we're trying to move away from using `mtd` in favour of `AnalysisDataService.retrieve()`